### DISCUSSION TITLE
docs support translation

### DISCUSSION THREAD

**Discussion by okxiaoliang4** [supabase/supabase#32975] [Feature Requests]
# Improve documentation

support translate

## Link 

[docs](https://supabase.com/docs)

## Describe the problem

The reading experience is not very good for non-native English speakers.

## Describe the improvement
Facilitate reading of docs by non-native English speakers.

## Additional context

**Comment by okxiaoliang4** [comment]
I want to translate to Chinese now, should i create a PR try to support i18n or fork the repo build a new one? I am trying to support i18n now. want some suggest

**Comment by Hallidayo** [comment]
Hi everyone, due to inactivity on this issue I've moved the issue over to discussions/enhancements.
